.changeset/two-lizards-stay.md

```diff
@@ -0,0 +1,5 @@
+---
+'@commercetools-uikit/filters': patch
+---
+
+add missing prop in filter prop list storybook documentation
```

Comment:
A change to the storybook does not necessitate a new release of the component does it?  I was under the impression that changesets are only necessary for changes that are included in the published packages.  Lmk if I'm wrong on that tho

Reply:
Correct! Sorry, merged before seeing the comment.